### DISCUSSION TITLE
Using simplePaginate to avoid SELECT COUNT(*)

### DISCUSSION THREAD

**Discussion by Piket564** [Power-Components/livewire-powergrid#475] [Ideas]
### Summary

Good Morning,
I'm using Powergrid with a 500k+ table, the record count takes like 7 to 8 second.
I want to remove that.

![image](https://user-images.githubusercontent.com/19176918/172890036-61b31f20-e3e0-4468-a71d-6175aa6acd17.png)

(Needed to obfuscate the tables.)

### Why is this needed?

To keep the refresh page time low.

### Suggested Solution(s)

Include in ->showRecordCount('min') simplePaginate to avoid make "SELECT COUNT(*) [...]"

### Related issue(s)/PR(s)

-

**Comment by luanfreitasdev** [comment]
It would be nice to add this possibility together
